### PR TITLE
Fix pack metadata files

### PR DESCRIPTION
**Pull request by gal-berger**
## Status
- [] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/24669

## Description
Bad contact details in the pack_metadata files.

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by KGausel** [comment]
Looks good to me

File: Packs/Cymulate/ReleaseNotes/1_0_4.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Cymulate
+- fix contact details in pack_metadata.json
```
Comment (yuvalbenshalom) [inline Packs/Cymulate/ReleaseNotes/1_0_4.md:3]:
```suggestion
- Fix contact details
```

File: Packs/Digital_Defense_FrontlineVM/ReleaseNotes/1_1_1.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Digital Defense FrontlineVM
+- fix contact details in pack_metadata.json
```
Comment (yuvalbenshalom) [inline Packs/Digital_Defense_FrontlineVM/ReleaseNotes/1_1_1.md:3]:
```suggestion
- Fix contact details
```

File: Packs/Humio/ReleaseNotes/1_0_1.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Humio
+- fix contact details in pack_metadata.json
```
Comment (yuvalbenshalom) [inline Packs/Humio/ReleaseNotes/1_0_1.md:3]:
```suggestion
- Fix contact details
```

File: Packs/Infinipoint/ReleaseNotes/1_0_1.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Infinipoint 
+- fix contact details in pack_metadata.json
```
Comment (yuvalbenshalom) [inline Packs/Infinipoint/ReleaseNotes/1_0_1.md:3]:
```suggestion
- Fix contact details
```

File: Packs/Silverfort/ReleaseNotes/1_0_2.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Silverfort
+- fix contact details in pack_metadata.json
```
Comment (yuvalbenshalom) [inline Packs/Silverfort/ReleaseNotes/1_0_2.md:3]:
```suggestion
- Fix contact details
```

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
looks good, added proposed fixes to RN

File: Packs/Cymulate/ReleaseNotes/1_0_4.md (hunk 1 above)
Comment (gal-berger) [reply Packs/Cymulate/ReleaseNotes/1_0_4.md]: fixed

File: Packs/Humio/ReleaseNotes/1_0_1.md (hunk 3 above)
Comment (gal-berger) [reply Packs/Humio/ReleaseNotes/1_0_1.md]: fixed.

File: Packs/Silverfort/ReleaseNotes/1_0_2.md (hunk 5 above)
Comment (gal-berger) [reply Packs/Silverfort/ReleaseNotes/1_0_2.md:3]: fixed.

File: Packs/Infinipoint/ReleaseNotes/1_0_1.md (hunk 4 above)
Comment (gal-berger) [reply Packs/Infinipoint/ReleaseNotes/1_0_1.md:3]: fixed

**Review by yuvalbenshalom — APPROVED** [review]

File: Packs/illuminate/ReleaseNotes/1_0_1.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### ￿illuminate
```
Comment (ShahafBenYakir) [inline Packs/illuminate/ReleaseNotes/1_0_1.md:2]:
```suggestion
##### illuminate
```

Comment (gal-berger) [reply Packs/illuminate/ReleaseNotes/1_0_1.md:2]: fixed

**Pull request merged**